jobs/integration/conftest.py

```diff
@@ -137,6 +140,17 @@ async def _load(self):
         self.charm_channel = request.config.getoption("--charm-channel")
         self.vault_unseal_command = request.config.getoption("--vault-unseal-command")
 
+    def juju_base(self, series):
+        """Retrieve juju 3.1 base from series."""
+        if self.juju_version < (3, 1):
+            return f"--series={series}"
+        mapping = {
+            "bionic": "ubuntu@18.04",
+            "focal": "ubuntu@20.04",
+            "jammy": "ubuntu@22.04",
+        }
+        return f"--series={mapping[series]}"
```

Comment:
i need to look at this again, it may be
```suggestion
        return f"--base={mapping[series]}"
```